libraries/cli/include/cli/default_config.hpp

```diff
@@ -16,6 +16,8 @@ const char *default_json = R"foo({
   "network_sync_level_size": 10,
   "network_packets_processing_threads": 14,
   "network_peer_blacklist_timeout" : 600,
+  "network_peer_light_node_timeout" : 600,
+  "light-node-history" : 0,
```

Comment:
naming

Reply:
I think "light node history" should be a const number and is not changeable. 
Maybe you want add in config for testing to adjust a good number, and remove it later.